### PR TITLE
README: Update getting started section point 6 in the README.md

### PR DESCRIPTION
**Pull request by rpkoller**
At first i wanted to clarify the details about how to test an add-on locally. I've tried that for the first time today. i've read the documentation on ddev.readthedocs as well as the readme for this repo but each of them wasn't that clear. @rfay helped to clarify. so i thought it would make sense to clarify the process within the addon-template readme as well. aside that i've suggested one or two more details. definitely not perfect and might need some proof reading from a native speaker but i hope it might provide some value and a cue how to improve the paragraph

### CODE REVIEW

**Review by rfay — CHANGES_REQUESTED** [review]
This looks fine, but seems like you did things to the numbering that you didn't want to do. Probably should be sequential numbers.

**Comment by rpkoller** [comment]
woops. THAT i didn't want to. i've added a few returns to separate the old part from the new part while editiing. that shifted the numbering to the order you see right now. but github wasn't smart enough to revert the number when the returns got removed. but i haven't noticed that the new numbering remained. i've simply assumed it would switch back and havent checked. gotta see how i am able to update the commit.

**Review by rfay — APPROVED** [review]
Thanks! Let me know when to pull this.

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ A repository like this one is the way to get started. You can create a new repo
...
+6. Update `tests/test.bats` to provide a reasonable test for your repository. Tests are triggered either manually with `bats tests`, automatically on every push to the repository or periodically each night. Please make sure to attend to test failures when they happen. Others will be depending on you. `bats` is a simple testing framework that just uses `bash`. To run a Bats test locally, you have to [install bats-core](https://bats-core.readthedocs.io/en/stable/installation.html) first. Then you download your add-on, and finally run `bats ./tests/test.bats` within the root of the uncompressed directory. To learn more about Bats see the [documentation](https://bats-core.readthedocs.io/en/stable/).
```
Comment (rpkoller) [inline README.md:38]: The only detail i wonder rereading my suggestion. Would it make sense to make that part more clear

>  Tests are triggered either manually with `bats tests`

and change it to 

Tests are triggered by manually executing `bats ./tests/test.bat` 

`bats tests`is sort of unclear. with the new suggestion it is clarified that you are manually executing a terminal command?

Comment (rfay) [reply README.md:38]: That would be fine.

**Pull request merged**